src/python/k4a/docs/examples.md

```diff
@@ -0,0 +1,65 @@
+# Example Programs Using Azure Kinect Python API(K4A)
+
+The examples/ folder contains Python programs to help guide users in using the
+Azure Kinect Python API effectively.
+
+## Example Prerequisites
+
+The following are required in order to run the Python K4A examples.
+
+1. Python (>= 3.6)
+
+2. An internet connection to download required Python packages.
+
+3. The Python K4A package has been installed. To install this package in your
+   python environment, get the K4A wheel file (.whl) and "pip install <*.whl>".
+   The wheel file can be created from source using the instructions in 
+   [building](./building.md).
+
+4. An attached Azure Kinect device.
+
+Each example has additional python package requirements that are specified below.
+
+## Running Examples
+
+### the_basics.py
+
+A simple program to showcase the basic functions in the Azure Kinect API.
+
+Additional Prerequisites: None
+
+To run, open a command terminal and type: python the_basics.py
+The expected output is a print out on the terminal of the device, capture, and image properties.
+
+### image_transformations.py
```

Comment:
both image_transformations and simple_viewer are using plt.show() commands, which requires a GUI component. I had to install tkinter using `sudo apt install python3-tk` for this to work

Reply:
Added tkinter as a requirement.